### ISSUE TITLE
Incorrect accuracy for ImageNet pretrain model

### ISSUE THREAD

**Issue by youkaichao** [pytorch/vision#1114]
In the [documentation](https://pytorch.org/docs/stable/torchvision/models.html#classification), it says resnet50 has the error rate of `23.85`, that is, the accuracy of `76.15`. However, after running the test script [here](https://github.com/pytorch/examples/blob/master/imagenet/main.py), I get the accuracy of `76.13`.

Other models are not tested. But obviously, there are inconsistencies here.

**Comment by fmassa** [comment]
I've just re-run this model using the reference scripts and I indeed got the same results as you, `76.13`. I've tried running with and without `benchmark` mode in cudnn, as well as without cudnn, and all gave the same result.

The question now is if this was a small typo when @colesbury added the numbers in https://github.com/pytorch/vision/pull/94, or if there are some small numerical differences between the implementation somewhere that affects the results?

cc @soumith
